docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,727 @@
+    "ports":[
+      { "name": "https",
+        "containerPort": 443,
+        "hostPort": 443},
+      { "name": "local",
+        "containerPort": 8080,
+        "hostPort": 8080}
+        ],
+    "volumeMounts": [
+        { "name": "srvkube",
+        "mountPath": "/srv/kubernetes",
+        "readOnly": true},
+        { "name": "etcssl",
+        "mountPath": "/etc/ssl",
+        "readOnly": true},
+      ]
+    }
+],
+"volumes":[
+  { "name": "srvkube",
+    "hostPath": {
+        "path": "/srv/kubernetes"}
+  },
+  { "name": "etcssl",
+    "hostPath": {
+        "path": "/etc/ssl"}
+  },
+]
+}}
+```
+
+The `/etc/ssl` mount allows the apiserver to find the SSL root certs so it can
+authenticate external services, such as a cloud provider.
+
+The `/srv/kubernetes` mount allows the apiserver to read certs and credentials stored on the
+node disk.
+
+Optionally, you may want to mount `/var/log` as well and redirect output there.
+
+#### Starting Apiserver
+Place the completed pod template into the kubelet config dir
+(whatever `--config=` argument of kubelet is set to, typically
+`/etc/kubernetes/manifests`).
+
+Next, verify that kubelet has started a container for the apiserver:
+```
+$ sudo docker ps | grep apiserver:
+5783290746d5        gcr.io/google_containers/kube-apiserver:e36bf367342b5a80d7467fd7611ad873            "/bin/sh -c '/usr/lo'"    10 seconds ago      Up 9 seconds                              k8s_kube-apiserver.feb145e7_kube-apiserver-kubernetes-master_default_eaebc600cf80dae59902b44225f2fc0a_225a4695   ```
+```
+
+Then try to connect to the apiserver:
+```
+$ echo $(curl -s http://localhost:8080/healthz)
+ok
+$ curl -s http://localhost:8080/api
+{
+  "versions": [
+    "v1beta3",
+    "v1"
+  ]
+}
+```
+
+If you have selected the `--register-node=true` option for kubelets, they will now being self-registering with the apiserver.
+You should soon be able to see all your nodes by running the `kubect get nodes` command.
+Otherwise, you will need to manually create node objects.
+
+### Scheduler
+
+*TODO*: convert to version v1.
+
+Complete this template for the scheduler pod:
+```
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"kube-scheduler"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "kube-scheduler",
+    "image": "$SCHEDULER_IMAGE",
+    "command": [
+                 "/bin/sh",
+                 "-c",
+                 "/usr/local/bin/kube-scheduler --master=127.0.0.1:8080"
+               ],
+    "livenessProbe": {
+      "httpGet": {
+        "path": "/healthz",
+        "port": 10251
+      },
+      "initialDelaySeconds": 15,
+      "timeoutSeconds": 15
+    },
+    }
+],
+}}
+```
+Optionally, you may want to mount `/var/log` as well and redirect output there.
+
+Start as described for apiserver.
+
+### Controller Manager
+To run the controller manager:
+  - select the correct flags for your cluster
+  - write a pod spec for the apiserver using the provided template
```

Comment:
Please update to "controller manager":
The line above should say "...pod spec for the controller manager..." (not apiserver).
&
The line below should say "start the controller manager pod" (not scheduler)